### PR TITLE
NEXUS-34974 Publish nxrm helm charts to Sonatype helm repo and Artifact Hub

### PR DESCRIPTION
**Pull request by userSeprid**
#### Description of Change
Updating Jenkins job in order to reorganize arifacts generation. [Jira ticket](https://issues.sonatype.org/browse/NEXUS-34974) for reference

#### Things to Do Before Submitting

* Have you signed the [Sonatype CLA](https://sonatypecla.herokuapp.com/sign-cla)?
* Have you added and run automated tests for your change? No changes in helm itself
  (See the [README](https://github.com/sonatype/helm3-charts/blob/main/README.md))
* Have you run `helm lint` on your change? No changes in helm itself

### CODE REVIEW

File: Jenkinsfile-Release [hunk 1]
```diff
@@ -90,8 +90,10 @@ dockerizedBuildPipeline(
     String version = verifyIndexYamlAndTarFile(params.chart)
 
     // Set Jira Fix Version
-    jira.versionPrefix = jiraVersionMappings[chart]
-    jiraSetFixVersion(jira, version)
+    if ('nexus-iq-server'.equals(params.chart)) {
```
Comment (jflinchbaugh) [inline Jenkinsfile-Release:93]: nit: this is groovy, so technically, you can safely use the '=' operator in place of `equals` method. (i missed this in a previous PR.)

File: Jenkinsfile-Release [hunk 2]
```diff
@@ -90,8 +90,10 @@ dockerizedBuildPipeline(
     String version = verifyIndexYamlAndTarFile(params.chart)
 
     // Set Jira Fix Version
-    jira.versionPrefix = jiraVersionMappings[chart]
```
[the remark this answers is not in the document]
Comment (userSeprid) [reply Jenkinsfile-Release:93]: Refactored

[the remark this answers is not in the document]
Comment (jflinchbaugh) [reply Jenkinsfile-Release:93]: I'm not seeing the commit here on this PR/branch yet.

[the remark this answers is not in the document]
Comment (userSeprid) [reply Jenkinsfile-Release:93]: @jflinchbaugh, had an issues, pushed again via cli, should be here for 100%

**Review by bobotimi — APPROVED** [review]

File: Jenkinsfile-Release (hunk 2 above)
[the remark this answers is not in the document]
Comment (jflinchbaugh) [reply Jenkinsfile-Release:93]: thank you. this looks great. i'll approve and merge.

**Review by jflinchbaugh — APPROVED** [review]
+1

**Pull request merged**